### PR TITLE
Feat/mpc whitelabel

### PR DESCRIPTION
**Pull request by lionellbriones**

### CODE REVIEW

File: src/views/Start/Start.vue [hunk 1]
```diff
@@ -18,17 +22,44 @@ import log from 'loglevel'
...
+      return (this.whiteLabel?.isActive && this.isCustomVerifier && this.dappName && this.dappUrl) || !this.whiteLabel.isActive || !this.whiteLabel
+    },
+  },
   async created() {
     try {
       const { loginProvider, state, skipTKey, ...rest } = this.$route.query
       const stateParams = JSON.parse(safeatob(state))
       log.info('logging in with', loginProvider, state, skipTKey, rest)
       const { whiteLabel, loginConfig = {} } = stateParams
+      this.iframeOrigin = getIFrameOriginObject()
```
Comment (chaitanyapotti) [inline src/views/Start/Start.vue:57]: cannot get this here.
send it from bottom while opening start popup

File: src/containers/Popup/PopupLogin/PopupLogin.vue [hunk 2]
```diff
@@ -22,37 +22,22 @@
...
-              </a>
-            </div>
-            <v-spacer></v-spacer>
-            <div v-if="!whiteLabel.isActive">
-              <div class="d-flex align-center mb-2">
-                <span class="text_2--text mr-1">{{ t('dappLogin.poweredBy') }}</span>
-                <img alt="Torus Logo" height="10" :src="getLogo.logo" />
+              <div class="caption text-right text_2--text">{{ t('dappLogin.selfCustodial') }}</div>
+              <div class="text-right">
+                <img alt="Web3Auth Logo" height="15" src="../../../assets/images/web3auth-dark.svg" />
```
[the remark this answers is not in the document]
Comment (lionellbriones) [reply src/containers/Popup/PopupLogin/PopupLogin.vue]: done

File: src/views/Start/Start.vue (hunk 1 above)
Comment (lionellbriones) [reply src/views/Start/Start.vue]: done

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**